programs/server/Server.cpp

```diff
@@ -823,6 +818,11 @@ try
             total_memory_tracker.setSampleMaxAllocationSize(server_settings.total_memory_profiler_sample_max_allocation_size);
     }
 
+    auto server_pool = std::make_unique<Poco::ThreadPool>(3, server_settings.max_connections);
```

Comment:
why was it moved?

Reply:
Bad merge with master, I did not notice it got moved.